backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/C_OrderLine_StepDef.java

```diff
@@ -531,6 +532,13 @@ private void validateOrderLine(@NonNull final I_C_OrderLine orderLine, @NonNull
 			assertThat(orderLine.getC_Tax_ID()).isEqualTo(tax.getC_Tax_ID());
 		}
 
+
+		final String userElementString2 = DataTableUtil.extractStringOrNullForColumnName(row, "OPT." + COLUMNNAME_UserElementString2);
+		if (de.metas.util.Check.isNotBlank(userElementString2))
+		{
+			assertThat(orderLine.getUserElementString2()).isEqualTo(userElementString2);
+		}
```

Comment:
what about the other user elements?

Reply:
As I wrote in the concept, the plan was to have one user element in the cucumbers and see if it propagates. However, I revertes the validation from cucumber feature since it was not working and there is no more time for cucumbers. I can as well delete this validation. To be added back when we have more time.

Reply:
agree, pls revert it

Reply:
same for other _StepDef(s)